setup.py

```diff
@@ -4,11 +4,11 @@
 #   ./venv/bin/pip install --editable .
 #   ./venv/bin/pip install --editable .[dev]  # with dev requirements, too
 
+import glob
 import os
 import sys
-import glob
 
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
```

Comment:
I'm guessing most of these changes are from a formatter (black?). We don't really enforce one at the moment, so can you revert these changes?